Sources/SwifterSwift/UIKit/UIImageExtensions.swift

```diff
@@ -32,6 +32,35 @@ public extension UIImage {
         return withRenderingMode(.alwaysTemplate)
     }
 
+    /// SwifterSwift: Average color for this image
+    var averageColor: UIColor? {
```

Comment:
I think we are missing a can import here? CoreImage?

Reply:
Does this look correct? I am unfamiliar with some of these issues, unfortunately